src/FavCard/FavCard.js

```diff
@@ -1,8 +1,18 @@
 import React from "react";
+import '../Smile/Smile.css'
+import { FaSmileBeam } from 'react-icons/fa'
+
+
+const FavCard = ({ madlib, favorited, removeFavorite, id }) => {
+
 
-const FavCard = ({ madlib, key}) => {
   return (
-    <h4>{madlib}</h4>
+    <div className="fav-card">
+      <h4>{madlib}</h4>
+      <button className="favorite-button" onClick={() => removeFavorite(id)}>
+          <FaSmileBeam className={favorited ? "favorited" : "unfavorited"}/>
```

Comment:
Very clean one-liner ternary! I always end up making these huge and sloppy-looking.